osbuild/objectstore.py

```diff
@@ -283,14 +283,20 @@ def init(self):
         self._root = self.store.tempdir(prefix="host")
 
         root = self._root.name
-        # Create a bare bones root file system
-        # with just /usr mounted from the host
+        # Create a bare bones root file system. Starting with just
+        # /usr mounted from the host.
         usr = os.path.join(root, "usr")
         os.makedirs(usr)
+        # Also add in /etc/containers, which will allow us to access
+        # /etc/containers/policy.json and enable moving containers (skopeo).
+        # https://github.com/osbuild/osbuild/pull/1410
```

Comment:
Hey, I wonder if we should also link to https://github.com/containers/image/issues/2157 here and maybe have a comment like: `# Remove this bind mount once https://github.com/containers/image/issues/2157 is fixed` or similar?
(given that the underlying issue here is that there is no split of defaults in /usr and user/admin config in /etc)

Reply:
I figured since I linked to this PR and in the first few comments above I linked to that issue they would find it, but I'm happy to add another comment too if others would find it useful. Please add a thumbs up or down on the above comment to vote!

Reply:
ok added to the most recent push.

Reply:
Thank you!

Reply:
Just one note about removing the bind mount - we discussed within the team and due to the guarantee of backward compatibility, we won't be able to remove this bind mount in the future without potentially breaking someone's manifest. This one of the reasons why we wanted to scope down to mount only the specific sub-directory in `/etc`.

Reply:
Yes, I understand it and it is a fair assumption.

The problem (as you mentioned) is that when using the host as the builroot, we do not control which versions of packages are installed on it. Therefore it is possible to break image builds which worked just by upgrading osbuild to a version which would again stop mounting `/etc/containers` from the host, without updating the rest of the host packages.

We currently do not have a good (or any) policy for making breaking changes, such as reverting this PR.